src/components/alt-button/FavoriteButton.tsx

```diff
@@ -84,7 +86,12 @@ const FavoriteButton = ({
   return (
     <div
       className={cn(
```

Comment:
okay, also something that we should consider doing at some point, but just food for thought in case you were interested in tackling this, is that this should actually render a <button /> element. in case you weren't aware, rendering a div here is not the best cause a screen-reader or web-driver cant interact with divs that dont have a "tabIndex" (buttons have tabIndex by default). Also it makes the `disabled` prop much cleaner cause buttons have a `disabled` prop, that will prevent `onClick` from triggering.